css/index.css

```
@@ -1,3 +1,387 @@
+  font: inherit;
+  vertical-align: baseline;
+}
+
+/* HTML5 display-role reset for older browsers */
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+
+body {
+  line-height: 1;
+}
+
+ol,
+ul {
+  list-style: none;
+}
+
+blockquote,
+q {
+  quotes: none;
+}
+
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: "";
+  content: none;
+}
+
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+  box-sizing: border-box;
+}
+
+html,
+body {
+  height: 100%;
+  font-family: "Titillium Web", sans-serif;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+  font-family: "Bangers", cursive;
+  letter-spacing: 1px;
+  margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+/* UTILITIES */
+
+/* adds a bottom border for certain elements */
+.bottom-line {
+  padding-bottom: 50px;
+  border-bottom: 1px solid black;
+}
+@media (max-width: 800px) {
+}
+
+/* adds a top space for certain elements */
+.top-space {
+  margin-top: 20px;
+}
+
+/* container to set width of page */
+.container {
+  max-width: 1024px;
+  margin: 0 auto;
+}
+@media (max-width: 800px) {
+  .container {
+    max-width: 800px;
+  }
+}
+
+@media (max-width: 500px) {
+  .container {
+    max-width: 500px;
+  }
+}
+
+/* TYPOGRAPHY */
+
+/* increase size h2 */
+h2.header-styles {
+  font-size: 1.3em;
+}
+
+header .half-column h1 {
+  font-size: 4.8em;
+  line-height: 0.9;
+}
+
+/* increase size of paragraphs */
+section .half-column p,
+section .third-column p {
+  font-size: 1.1em;
+  line-height: 1.2;
+}
+
+address p {
+  font-size: 1.1em;
+}
+
+@media (max-width: 500px) {
+  h2.header-styles {
+    font-size: 1em;
+  }
+  section .half-column p,
+  section .third-column p {
+    font-size: 1em;
+    line-height: 1;
+  }
+
+  address p {
+    font-size: 1em;
+  }
+}
+
+/* FLEX ROWS AND COLUMNS */
+
+.row {
+  max-width: 95%;
+  margin: auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+.half-column {
+  flex: 0 1 49.7%;
+}
+
+.third-column {
+  flex: 0 1 32%;
+}
+
+@media (max-width: 500px) {
+  .half-column,
+  .third-column {
+    flex: 0 1 100%;
+    margin: 9% 0 9%;
+  }
+}
+
+/* NAVIGATION */
+
+nav.row {
+  width: 1024px;
+  margin: 32px auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+nav a {
+  color: grey;
+  text-decoration: none;
+}
+
+@media (max-width: 800px) {
+  nav figure {
+    order: -1;
+    width: 100%;
+    margin-bottom: 5%;
+  }
+  nav figure img {
+    display: block;
+    margin: 0 auto;
+  }
+}
+@media (max-width: 500px) {
+  nav.row {
+    max-width: 100%;
+  }
+  nav a {
+    width: 100%;
+    border-top: 1px solid black;
+    border-bottom: 1px solid black;
+    padding: 18px 0 18px;
+    text-align: center;
+    font-size: 1.2em;
+  }
+}
```

Add this to your 500px media query
```suggestion
	nav a:last-child {
		border-bottom: 1px solid black;
	}
```
This change will fix your mobile navigation borders from overlapping